src/test/java/thrift/logic/commands/AddExpenseCommandTest.java

```diff
@@ -147,7 +147,7 @@ public void deleteTransaction(Transaction transaction) {
         }
 
         @Override
-        public void setTransaction(Transaction target, Transaction editedTransaction) {
+        public void setTransaction(Transaction target, Transaction updatedTransaction) {
```

Comment:
Please update this same method under `AddIncomeCommandTest.java` as well, thank you!